cvat/apps/engine/utils.py

```diff
@@ -110,17 +110,34 @@ def parse_specific_attributes(specific_attributes):
     } if parsed_specific_attributes else dict()
 
 
+def parse_exception_message(msg):
+    parsed_msg = msg
```

Comment:
Probably need to handle a case when msg is empty by a reason. Please resolve a conflict and see the case.